src/script/page/MainContent/MainContent.tsx

```diff
@@ -0,0 +1,160 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+import {CSSTransition, SwitchTransition} from 'react-transition-group';
+import {css} from '@emotion/core';
+import {registerStaticReactComponent, useKoSubscribableChildren} from 'Util/ComponentUtil';
+
+import {ContentViewModel, ContentState} from '../../view_model/ContentViewModel';
+import {t} from 'Util/LocalizerUtil';
+import Icon from 'Components/Icon';
+import AboutPreferences from './panels/preferences/AboutPreferences';
+import Collection from './panels/Collection';
+import AccountPreferences from './panels/preferences/AccountPreferences';
+import DevicesPreferences from './panels/preferences/devices/DevicesPreferences';
+import OptionPreferences from './panels/preferences/OptionPreferences';
+import {ConversationState} from '../../conversation/ConversationState';
+import AVPreferences from './panels/preferences/AVPreferences';
+import {container} from 'tsyringe';
+import {ClientState} from '../../client/ClientState';
+import {UserState} from '../../user/UserState';
+
+type LeftSidebarProps = {
+  contentViewModel: ContentViewModel;
+  conversationState?: ConversationState;
+};
+const Animated: React.FC = ({children, ...rest}) => {
+  return (
+    <CSSTransition classNames="slide-in-left" timeout={{enter: 500}} {...rest}>
```

Comment:
classNames or className? 🤔

Reply:
Keen eyes :eye: 
It is classname**s** in this case, it's a react-transition-group props (see https://reactcommunity.org/react-transition-group/css-transition#CSSTransition-prop-classNames) ;)